client/lib/sites-list/index.js

```diff
@@ -32,7 +32,6 @@ module.exports = function() {
 					break;
 				case InvitesActionTypes.INVITE_ACCEPTED_SUCCESS:
 				case 'RECEIVE_DISCONNECTED_SITE':
-				case 'RECEIVE_DELETED_SITE':
```

Comment:
the `'RECEIVE_DELETED_SITE'` case is already defined 10 lines before with a `break;` statement so this one can't ever be reach:
```
case 'RECEIVE_DELETED_SITE':
	_sites.removeSite( action.site );
	break;
```
Just fixing a mistake here.